python/python/glide/async_commands/core.py

```diff
@@ -2326,6 +2326,33 @@ async def zscore(self, key: str, member: str) -> Optional[float]:
             await self._execute_command(RequestType.ZScore, [key, member]),
         )
 
+    async def zdiffstore(self, destination: str, keys: List[str]) -> int:
+        """
+        Calculates the difference between the first sorted set and all the successive sorted sets at keys and stores the
+        difference as a sorted set to `destination`, overwriting it if it already exists. Non-existent keys are treated
+        as empty sets.
+        When in Cluster mode, all keys must map to the same hash slot.
```

Comment:
```suggestion
        Note: When in Cluster mode, all `keys` must map to the same `hash slot`.
```

Reply:
There is only one other doc that includes this warning about hash slots, but it doesn't include "Note: " or angled quotes around hash slot. Most of the other docs with a "keys" arg also don't put angled quotes around keys, the only one that does is `exists`. I'm fine with adding these changes but it seem likes Sho usually asks me to remove them to be consistent with the other docs. Let me know what you think

Reply:
https://github.com/aws/glide-for-redis/pull/1223/files#diff-37602a66f00251738190a1e0ebddd5d027022433028ab20ba728852a05abab7eR2187-R2188
Agree to discuss doc format with aws team

Reply:
Sure, for now I'll leave as is to stay consistent with the other python docs but I can update if we talk with them and agree to follow a different format than we have here

Reply:
Also see my related comment here: https://github.com/Bit-Quill/glide-for-redis/pull/258#discussion_r1586813045

Reply:
Updated now to align with the existing docs (see [here](https://github.com/aws/glide-for-redis/pull/1369#discussion_r1588228777) for the existing patterns that I've followed)